plugins/moduleRewrite.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const proxy = '-proxy';
+
+module.exports = function replaceImport(originalPath, callingFileName, options) {
+  /** remove extension to normalise the path */
+
+  if (originalPath === 'three') {
+    originalPath = `${process.env.MODULE_URL}three${proxy}.js`;
```

Comment:
This is not a good approach.

These cases must be auto detected so a new module addition does not require us to change any code.